test/runtests.jl

```diff
@@ -1,6 +1,20 @@
 # LibFTD2XX.jl
+#
+# By Reuben Hill 2019, Gowerlabs Ltd, reuben@gowerlabs.co.uk
+#
+# Copyright (c) Gowerlabs Ltd.
 
 using LibFTD2XX
-using Compat
-using Compat.Test
 
+include("util.jl")
+
+numdevs = LibFTD2XX.createdeviceinfolist()
+if numdevs > 0
+  @info "found $numdevs devices. Running hardware tests..."
```

Comment:
Run no hardware tests anyway?

Reply:
The "no hardware" tests currently _require_ that there be FTD2XX no hardware present. They fail if it's present.